.coveragerc

```diff
@@ -649,6 +649,7 @@ omit =
     homeassistant/components/volvooncall/*
     homeassistant/components/w800rf32/*
     homeassistant/components/waqi/sensor.py
+    homeassistant/components/warmup4ie/climate.py
```

Comment:
When I said rename, I meant it everywhere, not just in the comment…

Reply:
Okay, sorry. I have renamed the component to "warmup".